### PR TITLE
Adding task to create github issue when automation pipeline fails

### PR DESCRIPTION
**Pull request by ameyapat**
## Proposed changes

Adding a task in automation pipeline to create a Github issue in the repo when the pipeline fails.
If scheduled automation fails, an issue will be created in the git repo and can be triaged accordingly.

## Type of change

- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [x] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [x] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: azure_pipelines/automation.yml [hunk 1]
```diff
@@ -98,4 +98,50 @@ jobs:
...
+
+        github = github3.GitHub()
+        github.session.auth = HTTPHardcodedHeaderAuth()
+        github_org, repo = "$(Build.Repository.Name)".split("/")
+        
+        # Get the members of AppleIdentityTeam in AzureAD organization 
+        apple_id_team = github.organization('AzureAd').team(3752064)
+        members = list(apple_id_team.members(role="all"))
+        
+        # Pick random member from team to assign issue to
```
Comment (oldalton) [inline azure_pipelines/automation.yml:136]: :)

Comment (ameyapat) [reply azure_pipelines/automation.yml:136]: Github allows assigning a team as reviewers on PRs but doesn't allow assigning issue to a team :(

Comment (ameyapat) [reply azure_pipelines/automation.yml:136]: I guess it doesn't matter to whom the issue is assigned to, OCE will assign it to themselves during triage anyway

Comment (oldalton) [reply azure_pipelines/automation.yml:136]: Agreed

**Pull request merged**